### PR TITLE
Adding time option for the Prompt Format

### PR DESCRIPTION
**Pull request by gustavo-castro**
## Description
I added a date option for the prompt (as asked for in #580 ). To use it, you just have to add "\\t" to the Prompt Format option.

## Checklist

- [x] I've added this contribution to the `changelog.md`.
- [x] I've added my name to the `AUTHORS` file (or it's already there).

### CODE REVIEW

File: pgcli/main.py [hunk 1]
```diff
@@ -491,6 +494,8 @@ def run_cli(self):
                 else:
                     query = self.execute_command(document.text, query)
 
+                self.now = datetime.datetime.today()
```
Comment (gustavo-castro) [inline pgcli/main.py]: I tried it at first, but the problem is the program calls get_prompt for every single action that is done (typing one letter, erasing one letter, etc), so it keeps updating the timer every single time get_prompt is called.

File: pgcli/main.py [hunk 2]
```diff
@@ -11,6 +11,7 @@
 import shutil
 import functools
 import humanize
+import datetime
```
Comment (gustavo-castro) [inline pgcli/main.py]: Going to change it.

File: pgcli/main.py (hunk 1 above)
Comment (j-bennet) [inline pgcli/main.py]: Ah, good point! In this case it's a sensible choice.

**Comment by gustavo-castro** [comment]
Sorry about the question, I'm quite new to open-source contributing, but I guess I should take a look at the fact that one check has failed? I actually created it in the european format, but it seems this can be easily fixed, there are parameters that actually do this, I will change it too.

**Comment by j-bennet** [comment]
@gustavo-castro Oh good! I only saw the initial `'%d/%m/%y %H:%M:%S'`. Yes, the `'%x %X'` is better.

Once you add your contribution to AUTHORS and changelog, this PR should be good to merge.

Thank you!

**Comment by j-bennet** [comment]
@gustavo-castro Oh, and the tests fail because you added your name to changelog, but it refers to a missing link, which should go in the end of changelog file, like this:

```
.. _`Gustavo Castro`: https://github.com/gustavo-castro
```

**Comment by j-bennet** [comment]
@gustavo-castro, you are so close. :) Just add that missing link to the changelog.

**Comment by gustavo-castro** [comment]
@j-bennet Thanks for the help and the incentive! :) I didn't have access to my home computer during the day (european time), I was intending to finish it earlier.

**Comment by j-bennet** [comment]
Perfect. :) Merging.

Thank you for the PR, and congratulations on your first contribution to `pgcli`!

🍬

**Pull request merged**